src/query/storages/stage/src/row_based_file/serialize_processor.rs

```diff
@@ -27,18 +29,20 @@ use crate::row_based_file::buffers::FileOutputBuffers;
 
 pub(super) struct SerializeProcessor {
     output_format: Box<dyn OutputFormat>,
+    ctx: Arc<dyn TableContext>,
 }
 
 impl SerializeProcessor {
     pub(super) fn try_create(
         input: Arc<InputPort>,
         output: Arc<OutputPort>,
         output_format: Box<dyn OutputFormat>,
+        ctx: Arc<dyn TableContext>,
     ) -> Result<ProcessorPtr> {
```

Comment:
```suggestion
impl SerializeProcessor {
    pub(super) fn try_create(
        ctx: Arc<dyn TableContext>,
        input: Arc<InputPort>,
        output: Arc<OutputPort>,
        output_format: Box<dyn OutputFormat>,
    ) -> Result<ProcessorPtr> {
```

Reply:
It's always a better practice to place 'ctx' as the first parameter.

Reply:
ok